### PR TITLE
Add `/docs` prefix onEnter keydown

### PR DESCRIPTION
**Pull request by itsdillon**
Add `/docs` as a prefix in the `onKeyDown` useCallback function during the `Enter` case.

### CODE REVIEW

File: motif-docs/components/app/app-search.mdx [hunk 1]
```diff
@@ -229,7 +229,7 @@ dispatch({ type: 'SET_RESULTS', results: [] })
 break
 case 'Enter': {
 dispatch({ type: 'SET_RESULTS', results: [] })
-window.open(state.results[state.selectedIndex]?.path, '\_self')
+window.open('/docs' + state.results[state.selectedIndex]?.path, '\_self')
```
Comment (flybayer) [inline motif-docs/components/app/app-search.mdx]: is there a way to do this without hard coding it here? To make it more robust

Comment (v3dillon) [reply motif-docs/components/app/app-search.mdx]: The issue is we're setting the `basePath` to `/docs` in `next.config.js`. 

So if we prefix the return path with `/docs` from the `buildTree` function in `next.config`, then the relative path on the `<a />` element will be `/docs/docs/...`.

I pushed an alternative solution though. Let me know what you think.

**Review by flybayer — COMMENTED** [review]
Ok I like this better, **except** now I think the actual issue is using `<a>` in `<SearchResult>` instead of nextjs `<Link>`. Because Link automatically adds basepath.

**Comment by v3dillon** [comment]
> Ok I like this better, **except** now I think the actual issue is using `<a>` in `<SearchResult>` instead of nextjs `<Link>`. Because Link automatically adds basepath.

~~Ah very good point~~

Edit: Actually, this is the same issue because basepath is still `/docs`. I'm prepending the result object with the basepath so we don't have to hardcode `/docs` in the `onEnter` keydown event.

**Comment by flybayer** [comment]
@dillonraphael just pushed an elegant fix :)

**Pull request merged**